<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmOwner</class>
 <widget class="QMainWindow" name="FrmOwner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Owner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OWNER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>266</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="idOwnerLabel">
       <property name="text">
        <string>Id Owner</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Txt_id_owner"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Txt_tempatlahir"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tanggal lahir&lt;/p&gt;&lt;p&gt;(bln/tgl/thn)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelkelamin">
       <property name="text">
        <string>Kelamin</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="Cmb_kelamin">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>pria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>wanita</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="PB_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="PB_update">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="PB_del">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="PB_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="tempatlahirLabel">
       <property name="text">
        <string>Tempat lahir</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nama</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Txt_nama"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="Txt_telepon"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labeltelepon">
       <property name="text">
        <string>Telepon</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="Txt_tanggallahir"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
